renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -61,16 +60,9 @@ void BraveWalletRenderFrameObserver::DidCreateScriptContext(
           brave_wallet::features::kBraveWalletSolanaFeature) &&
       base::FeatureList::IsEnabled(
           brave_wallet::features::kBraveWalletSolanaProviderFeature)) {
-    if (!js_solana_provider_) {
-      js_solana_provider_ = JSSolanaProvider::Install(
-          dynamic_params.brave_use_native_wallet,
```

Comment:
do we create regardless `dynamic_params.brave_use_native_wallet` now?

Reply:
it would return early in above `if (!dynamic_params.brave_use_native_wallet) check`
and we shouldn't handle the liftetime ourselves, v8 will handle the lifetime of that object

Reply:
ok